src-graphql/client.js

```diff
@@ -22,15 +22,61 @@ export default class Client {
     });
   }
 
+  fetchAll(paginatedModels) {
```

Comment:
I don't think this should be exposed as a public method. It's an easy path for the library user to make some very generic and long-running calls.